traffic-generator/index.js

```diff
@@ -159,4 +168,43 @@ for (let i = 0; i < 2; i++) {
 }
 }, { scheduled: false });
 
-postPetsHighTrafficTask.start();
\ No newline at end of file
+postPetsHighTrafficTask.start();
+
+const lowTrafficPaymentTask = cron.schedule('* * * * *', () => {
+    const lowLoad = getRandomNumber(lowLoadMinRequests, lowLoadMaxRequests);
+    for (let i = 0; i < lowLoad; i++) {
+        console.log('send low load traffic payment: ' + (i + 1))
+        sleep(2 * 1000)
+        postPaymentData(100, `low-traffic-payment-${i + 1}`)
+            .catch(err => {
+                console.error("Failed to post /api/payments/owners/1/pets/1, error: " + (err.response ? err.response.data : err.toString()));
+
+            }); // Catch and log errors
+        axios.get(`${baseUrl}/api/payments/owners/1/pets/1`, { timeout: 10000 })
+            .catch(err => {
+                console.error(`${baseUrl}/api/payments/owners/1/pets/1, error: ` + (err.response ? err.response.data : err.toString()));
+            }); // Catch and log errors
+    }
+}, { scheduled: false });
+
+lowTrafficPaymentTask.start();
+
+const generateHighPaymentLoad = async () => {
```

Comment:
Suggest to remove the high payment load. We don't need this. It will generate a lot of traffic which incurs more cost.

Reply:
removed.